doc/manuals/admin/database_model.md

```diff
@@ -283,8 +283,9 @@ Fields:
     is updated each time a notification is sent, to avoid violating throttling.
 -   **throttling**: minimum interval between notifications. 0 or -1 means no throttling.
 -   **reference**: the URL for notifications, either HTTP or MQTT
--   **mqttTopic**: MQTT topic (only in MQTT notifications)
--   **mqttQoS**: MQTT QoS value (only in MQTT notifications)
+-   **topic**: MQTT topic (only in MQTT notifications)
+-   **qos**: MQTT QoS value (only in MQTT notifications)
```

Comment:
Taking into account this at src/lib/mongoBackend/dbConstants.h

```
#define CSUB_MQTTTOPIC               "topic"
#define CSUB_MQTTQOS                 "qos"
```

It seems `mqttQoS` and `mqttTopic` was a typo.

NTC (informative)